platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/Policy.java

```diff
@@ -83,6 +83,10 @@ public CollectionPermission getAllowedAttributePermissions() {
         return names;
     }
 
+    public Collection<ContextAttributeMapping> getAllowedAttributes() {
```

Comment:
We're leaking this collection here and I worry that we'll end up hitting issues because of it. In particular, it looks like it might be comparable to the bug that we needed to resolve for DDF-1542 ([this PR|https://github.com/codice/ddf/pull/223) where we might want to update these values dynamically, at least in the case of integration testing.

We should return an unmodifiable copy collection.